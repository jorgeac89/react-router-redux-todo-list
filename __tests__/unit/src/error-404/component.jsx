import React from 'react';
import { shallow } from 'enzyme';
import Error404 from 'error-404';

describe('Error404 component', () => {
  const Error404Component = Error404.component;

  it('renders correctly', () => {
    expect(shallow(<Error404Component />)).toMatchSnapshot();
  });
});
