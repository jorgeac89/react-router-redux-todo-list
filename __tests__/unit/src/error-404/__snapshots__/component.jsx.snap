// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 component renders correctly 1`] = `
<div
  className="d-flex flex-column justify-content-center text-center text-dark h-100"
>
  <h2
    className="display-2"
  >
    404
  </h2>
  <div
    className="display-3"
  >
    <i
      className="fas fa-dragon mx-3"
    />
    <i
      className="fas fa-cat mx-3"
    />
    <i
      className="fas fa-pastafarianism mx-3"
    />
  </div>
  <h3
    className="font-weight-light mt-3"
  >
    We have found these mythical creatures, but not the page you are looking for.
  </h3>
</div>
`;
